src/DynamoCore/Graph/Workspaces/PackageInfo.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Graph.Workspaces
+{
+    /// <summary>
+    /// Class containing info about a Dynamo package. 
+    /// Used for serialization.
+    /// </summary>
+    internal class PackageInfo
+    {
+        /// <summary>
+        /// Name of the package
+        /// </summary>
+        internal string Name { get; set; }
+
+        /// <summary>
+        /// Version of the package
+        /// </summary>
+        internal string Version { get; set; }
```

Comment:
is there a reason not to use a Version object?

Reply:
Using Version object now